lib/lists.nix

```
@@ -1462,6 +1462,40 @@ rec {
   */
   take = count: sublist 0 count;
 
+  /**
+    Return the last (at most) N elements of a list.
+
+    # Inputs
+
+    `count`
+
+    : Number of elements to take
```

```suggestion
    : Maximum number of elements to pick
```

- Result may be smaller
- No accidental reference to `take` here